client/client_test.go

```diff
@@ -16,16 +18,191 @@ func init() {
 	rand.Seed(time.Now().UnixNano())
 }
 
+const caCert = `
```

Comment:
nit: since we have tls certs in the separate directory `./test/mtls/` maybe we can re-use them here to avoid re-introducing other certs here?

Reply:
ok